libraries/Microsoft.Bot.Builder/BotFrameworkApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Bot.Builder
+{
+    /// <summary>
+    /// Maps various endpoint handlers for the <see cref="ServiceCollectionExtensions.AddBot{TBot}(IServiceCollection, Action{BotFrameworkOptions})">registered bot</see> into the request execution pipeline using the V4 protocol.
+    /// </summary>
+    public static class BotFrameworkApplicationBuilderExtensions
+    {
```

Comment:
This all seems like it belongs in an integration layer.

Reply:
I'm not sure I understand, let's make time to talk it through.